src/Evolution/Systems/Cce/NpValues.cpp

```diff
@@ -0,0 +1,54 @@
+// Distributed under the MIT License.
```

Comment:
we don't like the name `NpValues` @nilsdeppe interpreted it as non-polynomial...